src/styles/blocks/stars.scss

```diff
@@ -0,0 +1,16 @@
+.stars {
+  display: flex;
+
+  &__star {
+    width: 16px;
+    height: 16px;
+    background-image: url(../images/star.svg);
+    background-repeat: no-repeat;
+    background-position: center;
+    margin-right: 4px;
+
+    &:nth-child(-n+4) {
+      background-image: url(../images/star-active.svg);
+    }

```

Comment:
Modifiers from strs hw should work here too (stars--3). And try to use [cycles](https://sass-lang.com/documentation/at-rules/control/for)